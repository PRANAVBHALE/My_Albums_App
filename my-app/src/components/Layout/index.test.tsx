import { render } from "@testing-library/react";
import AppLayout from ".";

describe("Layout component", () => {
  it("should render and match snapshot", () => {
    let view = render(
      <AppLayout>
        <div>Hello</div>
      </AppLayout>
    );

    expect(view).toMatchSnapshot();
  });
});
