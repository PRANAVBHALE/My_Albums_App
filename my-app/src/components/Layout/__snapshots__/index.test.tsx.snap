// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component should render and match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <main
        class="ant-layout-content css-dev-only-do-not-override-15rg2km"
        style="display: flex; align-items: center; justify-content: center; text-align: center; min-height: 120px; line-height: 120px; color: rgb(255, 255, 255); background-color: rgb(54, 69, 79);"
      >
        <div>
          Hello
        </div>
      </main>
    </div>
  </body>,
  "container": <div>
    <main
      class="ant-layout-content css-dev-only-do-not-override-15rg2km"
      style="display: flex; align-items: center; justify-content: center; text-align: center; min-height: 120px; line-height: 120px; color: rgb(255, 255, 255); background-color: rgb(54, 69, 79);"
    >
      <div>
        Hello
      </div>
    </main>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
